### ISSUE TITLE
Attribute changes (e.g. unsubscribing) on customer.io are failing

### ISSUE THREAD

**Issue by atduarte** [gitpod-io/gitpod#5440]
### Bug description

As seen in the [customer.io Activity Logs](https://fly.customer.io/env/106338/activity_logs/identified?type=failed_attribute_change) there is a huge number of "Failed Attribute Change" entries. The affected users are unable to unsubscribe—even though the Gitpod dashboard may show they did—or update the name used in the emails. 

Example error message:
```json
{
  -unsubscribed: {
    action: "Update Marketing Preferences (63)"
    campaign: "signup event triggered campaign (6)"
    reason: "Variable 'event.unsubscribed' is missing"
  }
}
```

I'm unable to determine when this started as there is no way to order the logs list. Already contacted customer.io to get more information.

### Steps to reproduce

I assume this is happening to all new users. Given that:
1. Create an account
2. Unsubscribe
3. Check in customer.io that you don't show up as unsubscribed.

### Anything else?

We should stop sending marketing emails while we don't fix the underlying issue and update the affected users' `unsubscribed` state. /cc @mikenikles @ChristinFrohne @pawlean

**Comment by atduarte** [comment]
The cause seems to be that the "event triggered campaigns" in customer.io were not updated to take the introduction. of the `unsubscribed_*` attributes into consideration.

As such, the next steps are:
- [ ] Fix customer.io "event triggered campaigns"
- [ ] Check if the issue is solved
- [ ] Manually reimport data into customer.io (Note: Include email in order to fix #5441)

**Comment by atduarte** [comment]
Problem solved. Subscription status in customer.io are now all up to date.

We keep having logs for "Failed Attribute Change" when there's no `full_name` defined but that's not a problem. I will investigate with customer.io how to avoid it though.

We were unable to successfully export the email list, not fixing #5441. It's not as urgent.
